### ISSUE TITLE
Fix Rubocop error output

### ISSUE THREAD

**Issue by rglassett** [appacademy/dotfiles#5]
Either disable Rubocop in Atom or figure out how to prevent it from opening Chrome dev tools.

**Comment by davidrunger** [comment]
Are you referring to the ENOENT error? I just tried `gem uninstall rubocop` (and then `Y` to remove the rubocop binary as well), and then re-installed `gem install rubocop`, and it seems to have worked.

Also, in the AtomLinter package settings, you can turn off the obnoxiously fluorescent line highlighting and change the warnings to just be dots in the gutter.

**Comment by davidrunger** [comment]
I think CJ and I got rubocop working properly on (almost) all of the pairing stations yesterday morning. On those where rubocop is not working, it is at least no longer raising error messages, just failing silently.

As for making sure that Rubocop works on our system images in the future, I added a note (a63ec23441f4ec610e852636f7003cb3151360cc) to the README to check that Rubocop is actually working before we clone that image onto the pairing stations.
